index.html

```diff
@@ -0,0 +1,371 @@
+						<ul class="nav justify-content-end">
+							<li class="nav-item">
+								<a class="nav-link text-light" href="#"
+									><i class="fa-brands fa-facebook-f"></i
+								></a>
+							</li>
+							<li class="nav-item">
+								<a class="nav-link text-light" href="#"
+									><i class="fa-brands fa-twitter"></i
+								></a>
+							</li>
+							<li class="nav-item">
+								<a class="nav-link text-light" href="#">English</a>
+							</li>
+							<li class="nav-item">
+								<a class="nav-link text-light" href="#">My Page</a>
+							</li>
+							<li class="nav-item">
+								<a class="nav-link text-light" href="#">Log out</a>
+							</li>
+						</ul>
+					</div>
+
+					<nav class="navbar navbar-expand-sm">
+						<div class="container-fluid bg-white">
+							<a class="navbar-brand text-dark" href="index.html"
+								><img
+									class="victoria-logo"
+									src="./asset/images/victoria-university-logo.png"
+									alt="Victoria-university-logo" />
+							</a>
+
+							<ul
+								class="navbar-nav conatiner-fluid d-flex justify-content-between align-items-center">
+								<li class="nav-item mx-4" id="desktop-nav">
+									<a class="nav-link text-dark" href="./index.html">Home</a>
+								</li>
+								<li class="nav-item mx-4" id="desktop-nav">
+									<a
+										class="nav-link text-dark"
+										id="desktop-nav"
+										href="./about.html"
+										>About</a
+									>
+								</li>
+								<li class="nav-item mx-4" id="desktop-nav">
+									<a class="nav-link text-dark" href="#">Program</a>
+								</li>
+								<li class="nav-item mx-4" id="desktop-nav">
+									<a class="nav-link text-dark" href="#">Sponsor</a>
+								</li>
+								<li class="nav-item mx-4" id="desktop-nav">
+									<a class="nav-link text-dark" href="#">Log out</a>
+								</li>
+								<li class="nav-item mx-4 border border-5 border-danger">
+									<a class="nav-link text-center text-danger" href="#"
+										>Our legacy</a
+									>
+								</li>
+							</ul>
+						</div>
+					</nav>
+				</div>
+			</header>
+
+			<main>
+				<!--Intro section -->
+				<section class="hero-container">
+					<div class="hero-content py-5 px-3">
+						<!-- <div class="hero-title"> -->
+						<h1 class="h1 text-danger mt-5 mb-4">
+							VICTORIA UNIVERSITY ALUMNI STUDENT SUMMIT 2023
+						</h1>
+
+						<p class="textColor border border-5 border-white p-2">
+							The university of victoria alumni Summit annually brings together
+							students, faculty, alumni and professionals to hear from
+							world-renowned industry leaders on pressing topics and trends that
+							are shaping the future of the business and will be conduct on a.
+						</p>
+						<div class="hero-event-date">
+							<h2 class="textColor mt-5">2023.08.26(MON) ~ WEN(28)</h2>
+							<p class="textColor">
+								@Victoria university,Kampala uganda main hall
+							</p>
+						</div>
+					</div>
+				</section>
```

Comment:
[REQUIRED] Could you fix the top section of your homepage? It should contain a top section with a column that includes a small line of text, the page title, a block of text, a medium line of text in bold letters, and a smaller line of text. One of the aims of this project is to teach you to follow client requirements.